config/vufind/Blender.ini

```diff
@@ -0,0 +1,226 @@
+; Blender configuration. Note that all backends enabled here must have their own
+; configuration set up properly first, i.e. VuFind must be configured to use each
+; individual backend first before Blender can be used.
+
+; Enabled backends and their labels. A label is a translatable string that is
+; displayed in the results list for each result. Use an empty string for no label.
+; Backend order defines the order the results are fetched from each backend.
+[Backends]
+Solr = "Local Index"
+Primo = "CDI"
```

Comment:
What does "CDI" mean? We should perhaps think about what the default configuration should look like here, since we probably don't want it to be possible for live VuFind URLs to trigger access to invalid backends via default configuration.

Reply:
That's "Central Discovery Index". I think this will be customized anyway, but you're absolutely right. I think the backends need to be commented out by default. We could also include Solr + Search2 as an example.